### PR TITLE
i18n prep: i18n-friendly string construction (SC-1636)

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?

This changes all of our ad-hoc string construction to be more friendly to i18n. Goals are:
- Replace constructed strings with full sentence strings of each possibility
    - Before: “hello {}”.format(“world” if True else “universe”)
    - After: “hello world” if True else “hello universe”
- Replace all pluralization via formatting/concatenation with two separate strings for singular and plural forms
- Use named arguments in all string formats

## Test Steps
No functional change - everything should still pass.

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [ ] Yes
 - [x] No

### CODE REVIEW

File: features/unattached_commands.feature [hunk 1]
```diff
@@ -439,14 +439,14 @@ Feature: Command behaviour when unattached
         """
         Couldn't import the YAML module.
         Make sure the 'python3-yaml' package is installed correctly
-        and \/usr\/lib\/python3\/dist-packages is in yout PYTHONPATH\.
+        and \/usr\/lib\/python3\/dist-packages is in your PYTHONPATH\.
```
Comment (renanrodrigo) [inline features/unattached_commands.feature:442]: LOL :awkward-monkey-look:

**Review by renanrodrigo — APPROVED** [review]
One step closer

File: uaclient/messages/__init__.py [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+        return self.pluralize_fn(n)
+
+
+P = PluralizableString
+
+
+# This is a placeholder for ngettext
+# all instances of this will be replaced with ngettext
+def pluralize(singular: str, plural: str, n: int):
+    if n == 1:
```
[the remark this answers is not in the document]
Comment (orndorffgrant) [reply uaclient/messages/__init__.py:20]: For english this is generally correct. "I have 0 oranges" is right.

And for other languages, this gets replaced by ngettext, so I didn't worry about it :upside_down_face:.

I believe ngettext has parameters for different treatment of `n`, though I'm not sure exactly how they're configured.

[the remark this answers is not in the document]
Comment (orndorffgrant) [reply uaclient/messages/__init__.py:20]: Aha it is configured in the translation file:

For en_US there is a line:
```
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

and for pt_BR it is
```
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
```

[the remark this answers is not in the document]
Comment (renanrodrigo) [reply uaclient/messages/__init__.py:20]: TIL

**Review by lucasmoura — APPROVED** [review]

**Pull request merged**